docs/config/ping-federate-example.md

```diff
@@ -135,7 +101,9 @@ MIICnzCCAYcCBgFkqEAQCDANBgkqhkiG9w0BAQsFADATMREwDwYDVQQDDAhhbGZyZXNjbzA
 8. On the **Credentials** tab click **Configure Credentials** which will launch a new set of tabs for configuring the credentials. 
 
    #### Configuring credentials
-   1. On the **Digital Signature Settings** tab select your organization's certificate from the **Signing Certificate** dropdown. 
+   1. On the **Digital Signature Settings** tab
+    * select your organization's key pair and certificate from the **Signing Certificate** dropdown. 
+    * Tick the box below **Include the certificate in the signature <KeyInfo> element**.
```

Comment:
@jamalkm is this necessary? I thought this was only require for debugging purposes.

Reply:
No, it's not required.